expressEngine/cms/product/error.js

```diff
@@ -1,3 +1,4 @@
 export default {
   alreadyRegistered: 'You have already registered, Please sign in using provided email id and password',
+  noRecord: 'no record found',
```

Comment:
@aman-successive Please place noRecord in common cms.

Reply:
updated it